test_env_vars.yaml

```diff
@@ -74,10 +74,17 @@ cached_dependencies:
     https_url: "https://github.com/release-engineering/retrodep.git"
   # Repository that will be used for the test
   test_repo:
+    # Use local bare remote repository
+    # If this is used, the git urls below should be paths and should be available for the workers
+    use_local: True
     # The tracked repository (remote)
-    ssh_url: "git@github.com:user/cachito_test_repo.git"
+    ssh_url: "./tmp/cachito-archives/integration-tests-git-repo.git"
     # The URL of the repository
-    https_url: "https://github.com/user/cachito_test_repo.git"
+    https_url: "./tmp/cachito-archives/integration-tests-git-repo.git"
+    # test repo user
+    git_user: "Arthur Dent"
```

Comment:
Are these values to be replaced by an actual user?
If so, you'd have to replace them with the actual ones at https://gitlab.cee.redhat.com/osbs/cachito-tests/-/tree/master/jenkins because any changes done to `test_env_vars.yaml` will affect how we run tests in jenkins.
(It's a different story that currently jenkins tests are broken due to unavailability of nexus host on the test environment)

Reply:
If we must make these changes temporarily until jenkins tests are fixed, I recommend creating a temp. `test_env_vars_local.yaml` instead of tempering with `test_env_vars.yaml`. Let me know your thoughts.

Reply:
> Are these values to be replaced by an actual user?

Not for local tests.

> If so, you'd have to replace them with the actual ones at https://gitlab.cee.redhat.com/osbs/cachito-tests/-/tree/master/jenkins

I don't think we should mix upstream with downstream discussions. You are right though. The work will need to be done in the Jenkins side.

> If we must make these changes temporarily until jenkins tests are fixed, I recommend creating a temp. test_env_vars_local.yaml instead of tempering with test_env_vars.yaml

I don't think the configuration file should reflect whatever is used in jenkins. Instead, this should be either a generic configuration file or one set to work with local test runs.

Reply:
I got rid of the url from my comment above. You're right about not mixing both discussions but if the changes proposed here could potentially break how we run tests in jenkins, it's important to acknowledge them here before we merge them.

I agree that the configuration file does not reflect whatever is used in jenkins but it is directly fed to jenkins as it is for running the tests. Hence, let's just keep in mind while changing anything in configurations here, that all necessary changes required in downstream jenkins configuration must be done simultaneously (as part of the same ticket).

Reply:
You are right. I made an additional change to make sure this patch will not break custom existing configuration files